docs/dev/restart_strategies.md

```diff
@@ -28,8 +28,9 @@ In case that the job is submitted with a restart strategy, this strategy overrid
 
 The default restart strategy is set via Flink's configuration file `flink-conf.yaml`.
 The configuration parameter *restart-strategy* defines which strategy is taken.
-Per default, the no-restart strategy is used.
-When checkpointing is activated and no restart strategy has been configured, the job will be restarted infinitely often.
+If checkpointing is not enabled, the "no restart" strategy is used.
+If checkpointing is activated and the restart strategy has not been configured, the fixed-delay strategy is used with 
```

Comment:
Should we say "If checkpointing is activated and no restart strategy has been explicitly configured, ..."